src/option_learning.py

```diff
@@ -16,6 +18,8 @@ def create_option_learner() -> _OptionLearnerBase:
         return _KnownOptionsOptionLearner()
     if CFG.option_learner == "oracle":
         return _OracleOptionLearner()
+    if CFG.option_learner == "neural":
+        return _NeuralOptionLearner()
```

Comment:
Writing this here cuz I can't write this below. @line 26: we should have _OptionLearnerBase inherit from abc.ABC. I had this in a previous commit but it got lost in the back and forth. The reason is: https://stackoverflow.com/questions/56008847/when-should-one-inherit-from-abc

Reply:
This would need to happen in a lot of places. It's really just a matter of convention so I don't mind if we want to not do it. But in either case, we should do it in a separate PR if we want to do it at all.

Reply:
thanks both! sounds good, yeah let's save it for another PR if we do want to do it.

Reply:
it's a good idea for sure, helps pylint be less confused too. see #340